doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,101 @@
+============================================
+Cross Validation
+============================================
+
+Cross Validation is a type of model validation technique for figuring out how the
```

Comment:
Could you put every sentence in a new line. This makes later patches easier to review as the diff is smaller

Reply:
"Cross validation (lower case v) is a technique to estimate an algorithm's performance on unseen data" would be a bit more concise

Reply:
Sorry for all the capitalizations (trying to get rid of that habit). I've tried to remove all capitalizations and make every sentence on a new line to reduce the diff.